pkg/controller/appmetricsservice/util.go

```diff
@@ -0,0 +1,36 @@
+package appmetricsservice
+
+import (
+	"fmt"
+	"strings"
+
+	metricsv1alpha1 "github.com/aerogear/app-metrics-operator/pkg/apis/metrics/v1alpha1"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func labels(cr *metricsv1alpha1.AppMetricsService, suffix string) map[string]string {
+	return map[string]string{
+		"app":     cr.Name,
```

Comment:
I need a static label on the route for the app cr. Can you add one for me please?